aep/general/0134/aep.md.j2

```diff
@@ -57,7 +59,31 @@ separator, `snake_case` is used.
 
 {% tab oas %}
 
-**Note:** OAS example not yet written.
+```yaml
+paths:
+  /v1/publishers/{publisher}/books:
+    patch:
+      summary: Update a book
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Book'
+          application/merge-patch+json:
+            schema:
+              $ref: '#/components/schemas/Book'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Book'
+```
+
+- The method **must** adhere to the behavior specified in [IETF RFC 7396 - JSON Merge Patch](https://datatracker.ietf.org/doc/html/rfc7396).
+- The method **must** support MIME types `application/merge-patch+json` to adhere to IETF RFC 7396, as well as `application/json` as a common generalization.
```

Comment:
I wonder if it is a good idea to allow "application/json". RFC7396 has certain requirements, e.g. how null values are processed, that could be disregarded if the body is sent as "application/json". That creates ambiguity which seems bad to me.

Reply:
my primary concern with not supporting `application/json` is that it's very common for users to expect try that first - I suppose they'll just get a 415 unsupported media type or something to that effect and could try again though.

Reply:
Must it support application/json? If this was a "may" then we'd be leaving it to the API implementer to decide if they want to allow this looser type + the default that application/json is a merge-patch.

Reply:
Apologies for the delay here. based on https://github.com/aep-dev/aeps/pull/206#discussion_r1720988023 and your feedback, I'll be removing the optional clause on `application/json` for now - Richard's feedback in the linked thread that we shouldn't make merge-patch the defacto standard here makes sense to me, especially I think we don't feel there's a perfect protocol today. 

I'll resolve this thread, but feel free to chime in further in the linked one!